src/main/java/io/aiven/kafka/auth/json/AivenAcl.java

```diff
@@ -156,6 +195,8 @@ public String toString() {
                 + ", operationRe=" + operationRe
                 + ", resourceRe=" + resourceRe
                 + ", resourceRePattern='" + resourceRePattern
+                + ", resourceLiteral='" + resourceLiteral
+                + ", resourcePrefix='" + resourcePrefix
```

Comment:
Missing closing quotes in string representation. But maybe it would be better to export only the field that has been set instead of all?

Reply:
> Missing closing quotes in string representation. But maybe it would be better to export only the field that has been set instead of all?
yes good idea :+1: